server/tests.go

```diff
@@ -16,9 +19,44 @@ func (ge *TestsEndpoint) AttachHandlers(r *mux.Router) {
 	r.HandleFunc("/ping", ge.ping)
 }
 
+func (ge *TestsEndpoint) checkExternal() bool {
+	isHealthy := true
+
+	if remote_signer.EnableRethinkSKS {
+		conn := database.GetConnection()
+
+		if !conn.IsConnected() {
+			isHealthy = false
+		}
+	}
+
+	if remote_signer.VaultStorage {
+		kb := vaultManager.MakeVaultManager(remote_signer.KeyPrefix)
+		health, err := kb.Client.Sys().Health()
```

Comment:
Eu acho melhor expor um VaultHealth no `VaultManager` ao invés de expor o Client, dado que expondo o client você pode mutar o valor de dentro do VaultManager por acidente.

Reply:
> Also, add tests for the file `tests.go`. The coverage shouldn't be reduced.

How can I increase the coverage if the test is a HTTP request?